adminapi/__init__.py

```diff
@@ -1,17 +1,6 @@
 """Serveradmin - adminapi
 
-Copyright (c) 2018 InnoGames GmbH
+Copyright (c) 2020 InnoGames GmbH
 """
 
-from adminapi.request import Settings
-
-
-# XXX Deprecated
-def auth(auth_token=None):
-    if auth_token:
-        Settings.auth_token = auth_token
-
-
-# XXX Deprecated
-def set_timeout(timeout, what='api'):
-    Settings.timeout = timeout
+VERSION = (1, 9, 3)
```

Comment:
Is it possible to bump version here as well, please?

Reply:
I increased the version by accident. This is fixed. This should be the only place where we have to bump the version for now.